lib/catalog/owid/catalog/datasets.py

```diff
@@ -41,7 +41,7 @@
 assert SUPPORTED_FORMATS[0] == PREFERRED_FORMAT
 
 # available channels in the catalog
-CHANNEL = Literal["garden", "meadow", "grapher", "backport", "open_numbers", "examples", "explorers"]
+CHANNEL = Literal["garden", "meadow", "grapher", "backport", "open_numbers", "examples", "explorers", "external"]
```

Comment:
Nit - We could add a constant listing all channels extracted from `Literal` type with
```
from typing import get_args
CHANNELS = get_args(CHANNEL)
```
which could be used to avoid enumerating all channels each time.

Reply:
Good idea. I've noticed we use `CHANNEL.__args__` in many places. So maybe it's better to replace all those with CHANNELS (but better in a separate PR), to avoid confusion.